pkg/expr/transform.go

```diff
@@ -151,9 +152,3 @@ func hiddenRefIDs(queries []Query) (map[string]struct{}, error) {
 	}
 	return hidden, nil
 }
-
-func (s *Service) decryptSecureJsonDataFn(ctx context.Context) func(ds *datasources.DataSource) (map[string]string, error) {
```

Comment:
these changes seem unrelated?

Reply:
Because `dataSourceService` was removed as a dependency of `Service`, now the pluginContextProvider will take care of this instead